test('Hello World', () => {
    
})


test('should faild', () => {
    throw new Error('Failure')
})

